### ISSUE TITLE
hCaptcha will not load when UV is iframed (about:blank)

### ISSUE THREAD

**Issue by itschasa** [titaniumnetwork-dev/Ultraviolet#128]
without about:blank cloak:
![image](https://github.com/titaniumnetwork-dev/Ultraviolet/assets/79016507/f61f3cce-fa4d-4f6b-a73b-d33fc5f0c62d)

with about:blank cloak (iframe):
![image](https://github.com/titaniumnetwork-dev/Ultraviolet/assets/79016507/ead47954-26fc-41af-b4c9-ae7786264ad8)

iframe to front end:
```html
<iframe src="https://******" style="height:100%; width: 100%; border: none; position: fixed; top: 0; right: 0; left: 0; bottom: 0; border: none" sandbox="allow-forms allow-downloads allow-modals allow-orientation-lock allow-pointer-lock allow-popups allow-popups-to-escape-sandbox allow-presentation allow-same-origin allow-scripts allow-top-navigation allow-top-navigation-by-user-activation"></iframe>
```
iframe for service worker to pickup:
```html
<iframe sandbox="allow-same-origin allow-downloads allow-scripts allow-forms allow-modals allow-orientation-lock allow-pointer-lock allow-presentation" srv="us1" src="https://******/'service'/*****"></iframe>
```

this probably isn't fixable, as no errors are appearing in console. most likely hcaptcha blocking when 2 iframes deep (3 iframes deep from its perspective (about:blank > uv > hcaptcha iframe))? not sure, but best to report it incase others have issues

recaptcha still works fine when about:blank-ed

**Comment by Percslol** [comment]
yeah this is a known issue, not sure how we would go about fixing it

probably going to keep this issue open though for further discussion

**Comment by itschasa** [comment]
> 
> 
> 
> 
> Which version are you using?
> 
> ![image](https://github.com/titaniumnetwork-dev/Ultraviolet/assets/93471589/205c47be-27cc-40a1-84f3-665f2d91950a)
> 
> you can try [https://hideip.network/](https://hideip.network/)

Did you try it whilst about:blank cloaking was enabled?

**Comment by itschasa** [comment]
> After testing, uv cannot rewrite iframe again

![image](https://github.com/titaniumnetwork-dev/Ultraviolet/assets/79016507/15f7b53e-1833-43b4-8ba7-70c6cd837567)

looks like UV can inject itself into an iframe whilst about:blank cloaked, but hcaptcha just refuses to load, most likely a security mesaure by hcaptcha, preventing captchas being done whilst 2 iframes deep already
